server/controllers/eventsController.js

```diff
@@ -2,7 +2,8 @@ const {Event} = require('../models/Event');
 
 
 const createEvent = async (req, res) => {
-    const eventForm = {title: 'Cyber Security', location: '456 A St BLVD', time_start: new Date(), time_end: new Date(), isVirtual: true } //temporary until react form is made
+    debugger;
```

Comment:
We should try to remove `debugger` and `console.log()` expressions before merges.